<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TicketSystem_Server</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>781</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>服务端消息: </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>805</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_func">
    <property name="title">
     <string>服务器操作</string>
    </property>
    <addaction name="action_start"/>
    <addaction name="action_stop"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_inquire">
    <property name="title">
     <string>机票查询</string>
    </property>
    <addaction name="action_inquireone"/>
    <addaction name="action_inquireall"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_show"/>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menu_func"/>
   <addaction name="menu_inquire"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_start"/>
   <addaction name="action_stop"/>
   <addaction name="action_inquireone"/>
   <addaction name="action_inquireall"/>
   <addaction name="action_exit"/>
  </widget>
  <action name="action_start">
   <property name="icon">
    <iconset resource="Icon.qrc">
     <normaloff>:/images/images/start.png</normaloff>:/images/images/start.png</iconset>
   </property>
   <property name="text">
    <string>开启服务器</string>
   </property>
  </action>
  <action name="action_stop">
   <property name="icon">
    <iconset resource="Icon.qrc">
     <normaloff>:/images/images/stop.png</normaloff>:/images/images/stop.png</iconset>
   </property>
   <property name="text">
    <string>关闭服务器</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="icon">
    <iconset resource="Icon.qrc">
     <normaloff>:/images/images/power-off.png</normaloff>:/images/images/power-off.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_inquireone">
   <property name="icon">
    <iconset resource="Icon.qrc">
     <normaloff>:/images/images/download.png</normaloff>:/images/images/download.png</iconset>
   </property>
   <property name="text">
    <string>查询指定航班</string>
   </property>
  </action>
  <action name="action_inquireall">
   <property name="icon">
    <iconset resource="Icon.qrc">
     <normaloff>:/images/images/download.png</normaloff>:/images/images/download.png</iconset>
   </property>
   <property name="text">
    <string>查询所有航班</string>
   </property>
  </action>
  <action name="action_show">
   <property name="text">
    <string>显示内容</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Icon.qrc"/>
 </resources>
 <connections/>
</ui>
